.jvmopts

```diff
@@ -0,0 +1,3 @@
+-Xms1G
+-Xmx4G
+-XX:+UseG1GC
```

Comment:
Is this needed for 17?

Reply:
The build was hanging on the `fastOptJS` step, usually it's a memory problem. These flags help.